app/serializers/activitypub/actor_serializer.rb

```diff
@@ -119,6 +119,24 @@ def moved_to
   class CustomEmojiSerializer < ActivityPub::EmojiSerializer
   end
 
+  class TagSerializer < ActiveModel::Serializer
+    include RoutingHelper
+
+    attributes :type, :href, :name
+
+    def type
+      'Hashtag'
+    end
+
+    def href
+      explore_hashtag_url(object)
```

Comment:
Not too sure statuses hashtags and account hashtags should be considered as two entirely different things?